sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseChannel.java

```diff
@@ -34,7 +35,9 @@
 import reactor.core.publisher.ReplayProcessor;
 
 import java.io.IOException;
+import java.nio.ByteBuffer;
 import java.time.Duration;
+import java.util.Map;
```

Comment:
Any reason map was added as an import? I'm not seeing it being used in new code.